server_admin/topics/users/proc-setting-password-user.adoc

```diff
@@ -19,8 +19,13 @@ If a user already has a password, it can be reset in the *Reset Password* sectio
 +
 NOTE: If *Temporary* is *ON*, the user must change the password at the first login. To allow users to keep the password supplied, set *Temporary* to *OFF.*  The user must click *Set Password* to change the password.
 +
-. Alternatively, you can send an email to the user that requests the user reset the password.  
+. Alternatively, you can send an email to the user that requests the user reset the password.
```

Comment:
I don't see Credential Reset in the old or new UI.  You might have a later build beyond 18,
However, I have the KC15 build for the old UI and I don't see this Credential Reset.

Reply:
I don't think this is very new, it's here in the old console:
![image](https://user-images.githubusercontent.com/51133/168998693-e299c6fd-05b1-42c9-b7eb-d06374fbd107.png)
And here in the new UI:
![image](https://user-images.githubusercontent.com/51133/168999077-0f17667e-9659-464f-8810-3e53a0c6ac23.png)
You have to have a user with an email of course otherwise where would you send the email to.

Reply:
Okay, I see. Thanks for clarifying.